client/my-sites/site-settings/manage-connection/site-ownership.jsx

```diff
@@ -229,7 +228,15 @@ class SiteOwnership extends Component {
 	}
 
 	render() {
-		const { siteId, siteIsConnected, siteIsJetpack, translate } = this.props;
+		const { canManageOptions, siteId, siteIsConnected, siteIsJetpack, translate } = this.props;
+
+		if ( canManageOptions === false ) {
+			return null;
+		}
+
+		if ( ! canManageOptions ) {
+			return this.renderPlaceholder();
+		}
```

Comment:
Separate `=== false` and `!` checks are always a bit ugly -- and not too intuitive: I wouldn't know off the top of my head what means what for `canManageOptions`. Given the `return` values, I assume one case is for when we don't have any user and/or site data at all yet (the `!` case), and hence want to show the placeholder.

Can we make that more evident (and get rid of the `=== false` check) by checking for e.g. `! siteId` (or something like that, provided that it's already available in `connect()`), and moving that check up?

I.e. something like

```js
if ( ! siteId ) {
	return this.renderPlaceholder();
}

if ( ! canManageOptions ) {
	return null;
}
```

Reply:
👍 Done in c068b1c460e619cefb2fefc3a7e0572290e653bf.